File: MPI/mpicheck/tests/functions/1.0/MPI_Type_indexed.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1(10)
       INTEGER var_2(10)
       INTEGER var_3
       INTEGER var_4
       INTEGER var_5
        call mpi_type_indexed(var_0, var_1, var_2, var_3, var_4, var_5)
       call pmpi_type_indexed(var_0, var_1, var_2, var_3, var_4, var_5)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Type_indexed.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    const int var_1[2];
    const int var_2[2];
    MPI_Datatype var_3;
    MPI_Datatype *var_4;
    int ret;
    /* calls */
    ret = MPI_Type_indexed(var_0, var_1, var_2, var_3, var_4);
    ret = PMPI_Type_indexed(var_0, var_1, var_2, var_3, var_4);
    return 0;
}
